docs/predictive-analytics/model-retraining/process.sql

```diff
@@ -0,0 +1,64 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create stream of predictions
+CREATE STREAM PREDICTED_WEIGHT(
+  "Fish_Id" VARCHAR KEY,
+  "Species" VARCHAR,
+  "Height" DOUBLE,
+  "Length" DOUBLE,
+  "Prediction" DOUBLE
+  )
+WITH(
+  KAFKA_TOPIC = 'kt.mdb.weight-prediction', 
+  VALUE_FORMAT = 'JSON'
```

Comment:
Need `PARTITIONS`, otherwise error (see other recipes for example)

```
Topic 'kt.mdb.weight-prediction' does not exist. If you want to create a new topic for the stream/table please re-run the statement providing the required 'PARTITIONS' configuration in the WITH clause (and optionally 'REPLICAS'). For example: CREATE STREAM PREDICTED_WEIGHT (`Fish_Id` STRING KEY, `Species` STRING, `Height` DOUBLE, `Length` DOUBLE, `Prediction` DOUBLE) WITH (KAFKA_TOPIC='kt.mdb.weight-prediction', KEY_FORMAT='KAFKA', PARTITIONS=2, VALUE_FORMAT='JSON');
```